worlds/rabi_ribi/client/client.py

```diff
@@ -622,10 +622,11 @@ async def rabi_ribi_watcher(ctx: RabiRibiContext):
             if cur_time - ctx.time_since_last_item_obtained > 7:
                 ctx.rr_interface.remove_exclamation_point_from_inventory()
 
-            if ctx.rr_interface.get_number_of_eggs_collected() >= 5:
-                ctx.finished_game = True
-                await ctx.send_msgs([{"cmd": "StatusUpdate", "status": ClientStatus.CLIENT_GOAL}])
-                
+            if ctx.slot_data:
+                required_egg_count = ctx.slot_data["required_egg_count"] if "required_egg_count" in ctx.slot_data else 5
```

Comment:
Suggestion, other games like Hatsune Miku have a command in the text client which displays how many of the McGuffins are needed to beat the game. If the required egg count can vary, maybe we should add that command.

Reply:
Going to log this as a separate task, as we don't have a command processor implemented in the client yet (I do have a stash with one for adding an improved `/get_logical_path` for UT)
Created #41 to track this.